runtime/shared_common/OSCache.hpp

```diff
@@ -61,7 +61,11 @@
 #define OSCACHE_LOWEST_ACTIVE_GEN 1
 
 /* Always increment this value by 2. For testing we use the (current generation - 1) and expect the cache contents to be compatible. */
+#ifdef J9VM_OPT_VALHALLA_NESTMATES
+#define OSCACHE_CURRENT_CACHE_GEN 37
+#else
 #define OSCACHE_CURRENT_CACHE_GEN 35
+#endif /* J9VM_OPT_VALHALLA_NESTMATES */
```

Comment:
@pshipton and I discussed this: the generation number does not need to be updated. We don't need to have a separate non-nestmate generation. The Java version (e.g. 8 or 11) is included in the name of the file used in addition to the generation so there won't be confusion between Java versions.

Reply:
Not incrementing the generation will cause a problem for any existing Java 11 shared caches, but since we don't have any regular builds of Java 11 yet, it seems the better choice to just go ahead without changing the cache generation.